docs/data/date-pickers/date-range-picker/BasicDateRangePicker.js

```diff
@@ -9,10 +9,11 @@ export default function BasicDateRangePicker() {
   const [value, setValue] = React.useState([null, null]);
 
   return (
-    <LocalizationProvider dateAdapter={AdapterDateFns}>
+    <LocalizationProvider
+      dateAdapter={AdapterDateFns}
+      localeText={{ start: 'Check-in', end: 'Check-out' }}
```

Comment:
I'm wondering if moving this `start` and `end` in the l10n was a good idea. This could be "Check-in" "Departure", "Arrival", "Begining", ... according to the context of the application

Maybe it is better to rename them `startLabel` since it is the label applied to the `TextField` for the start date

Reply:
Maybe we could add a `localeText` key on the components as well and if they are defined, it wrap the component with a context (with a small HOC).

Reply:
Is it used elsewhere ?
If not I indeed don't see the point of this prop in the 1st place

Reply:
My bad, it is used for the toolbar of the mobile date range picker when no value is selected